radis/test/lbl/test_base.py

```diff
@@ -855,10 +857,10 @@ def test_caching_noneq_params(verbose=True, plot=True, *args, **kwargs):
         assert isotope == existing_file_metadata["isotope"]
     if "number_of_lines" in existing_file_metadata:
         assert len(s.df0) == existing_file_metadata["number_of_lines"]
-    if "wavenumber_min" in existing_file_metadata:
-        assert s.df0["wav"].min() == existing_file_metadata["wavenumber_min"]
-    if "wavenumber_max" in existing_file_metadata:
-        assert s.df0["wav"].max() == existing_file_metadata["wavenumber_max"]
+    if "wavenum_min" in existing_file_metadata:
```

Comment:
Test should happen always. If key is not in metadata you should regenerate the cache